common/assets/scss/helpers/_typography.scss

```diff
@@ -4,3 +4,7 @@
 
 // Ensure font override is loaded
 @include _inter-ui-font-face();
+
+.govuk-secondary-text-colour {
```

Comment:
I'm not sure this is the correct place for this class. The Design System readme [suggests that helpers](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/README.md#helpers) "should not output any CSS".

Maybe this class would live better within [utilities](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/README.md#utilities) - "Utility classes – for example clearing floats, visually hiding content"